index.js

```diff
@@ -21,6 +21,8 @@ class DaikinCloudController extends EventEmitter {
      * @param {object} options.proxyDataDir Data directory to store certificates and other needed files, defaults to library root directory
      * @param {function} [options.logger=console.log] Logger function
      * @param {string} [options.logLevel=info] Loglevel to use - in fact only "debug" has a meaning to log some more details
+     * @param {object} options.communicationTimeout=10000 Timeout in ms for Requests & Responses to Cloud
```

Comment:
correct type is number
```suggestion
     * @param {number} options.communicationTimeout=10000 Timeout in ms for Requests & Responses to Cloud
```